root-packages/frida/build.sh

```diff
@@ -131,6 +138,7 @@ termux_step_post_configure() {
 
 termux_step_post_make_install () {
 	# Fixup installation location..
+	rm -rf "$TERMUX_PREFIX"/lib/python"${TERMUX_PYTHON_VERSION}"/site-packages/frida*
```

Comment:
Shouldn't this also be in pre_configure? I don't think removing these files in post_make_install affects anything for the build, since they are recreated two lines down